python/ray/data/_internal/util.py

```diff
@@ -206,3 +207,39 @@ def _check_import(obj, *, module: str, package: str) -> None:
             f"couldn't be imported. You can install '{package}' by running `pip "
             f"install {package}`."
         )
+
+
+def _resolve_local_scheme(path: str) -> str:
+    """Returns the resolved path if the given path following the local scheme.
+    Othewise, returns the path unchanged."""
+    if path.startswith(_LOCAL_SCHEME):
+        path = path[len(_LOCAL_SCHEME) :]
```

Comment:
Should we use URI parsing here instead of startswith?

Reply:
Sg, switched to uri parser.